ckanext/dcatde/harvesters/dcatde_rdf.py

```diff
@@ -396,6 +445,9 @@ def import_stage(self, harvest_object):
 
         # set custom field and perform other fixes on the data
         self._amend_package(harvest_object)
+        package = json.loads(harvest_object.content)
+        if package.get('owner_org') == None:
+            return False
```

Comment:
```suggestion
        # Ensure that the property "owner_org" is set for the dataset, if the option "remote_orgs" is set, otherwise skip the import.
        if self._get_remote_orgs_from_config(harvest_object.source.config) in ('only_local', 'create'):
            package = json.loads(harvest_object.content)
            if package.get('owner_org') == None:
                return False
```
And a Getter method could be added as follows, with can be called multiple from several places.
```
    @staticmethod
    def _get_remote_orgs_from_config(source_config):
        ''' Get remote_orgs from source '''
        if source_config:
            return json.loads(source_config).get(CONFIG_PARAM_REMOTE_ORGS)

        return ''
```
The definition of the constant `CONFIG_PARAM_REMOTE_ORGS` used in the example is mandatory.